assets/_locales/en/messages.json

```diff
@@ -3946,6 +3946,10 @@
     "message": "Help",
     "description": "Help text"
   },
+  "help_description": {
```

Comment:
Should i remove the help description or remove the description for it ?

Reply:
I think it's fine like this. We can ask the design team later if they want to change anything